picard/script.py

```diff
@@ -1375,16 +1388,14 @@ def func_ne_any(parser, x, *args):
 
 Returns `text` in title case (first character in every word capitalized).
 
+Example:
+
+    $set(album,$title(%album%))
 _Since Picard 2.1_"""
-))
+), debug_doc=True)
```

Comment:
Is debug_doc set by purpose?

Reply:
Nope, leftover from experiment, I'll clean this.